pkgs/stdenv/generic/setup.sh

```diff
@@ -7,6 +7,8 @@ if [ -n "${BASH_VERSINFO-}" ] && [ "${BASH_VERSINFO-}" -lt 4 ]; then
     exit 1
 fi
 
+set -x
```

Comment:
Isn't this covered by line 13?

Reply:
that is a leftover from my debugging. Sorry for leaving that!
For some reason I can't get it to work with NIX_DEBUG, so I just forcefully added it.
My bad. Removing it from the commit